plugwise/smile.py

```diff
@@ -47,6 +50,44 @@
 class SmileData(SmileHelper):
     """The Plugwise Smile main class."""
 
+    def update_for_cooling(self, devices: dict[str, DeviceData]) -> None:
+        """Helper-function for adding/updating various cooling-related values)."""
+        for _, device in devices.items():
+            # For Anna + cooling, modify cooling_state based on provided info by Plugwise
+            if self.smile_name == "Anna":
+                if device["dev_class"] == "heater_central" and self._cooling_present:
+                    device["binary_sensors"]["cooling_state"] = False
+                    if self._anna_cooling_active:
+                        device["binary_sensors"]["cooling_state"] = True
+
+                # Add setpoint_low and setpoint_high when cooling is enabled
+                if device["dev_class"] not in ZONE_THERMOSTATS:
+                    continue
+
+                if self.anna_cooling_enabled:
+                    sensors = device["sensors"]
+                    if self._sched_setpoints is None:
+                        sensors["setpoint_low"] = sensors["setpoint"]
+                        sensors["setpoint_high"] = float(40)
+                        if self._anna_cooling_active:
+                            sensors["setpoint_low"] = float(0)
+                            sensors["setpoint_high"] = sensors["setpoint"]
+                    else:
```

Comment:
Should we just set the else and ditch the else?

Reply:
Yes, updated.

Reply:
Updated: no, this does not work.
When there's no schedule active I get this:
```
                if self.anna_cooling_enabled:
                    sensors = device["sensors"]
>                   sensors["setpoint_low"] = self._sched_setpoints[0]
E                   TypeError: 'NoneType' object is not subscriptable
```